bower.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "esri-leaflet-heatmap-feature-layer",
-  "version": "v2.0.0-beta.1",
   "main": "dist/esri-leaflet-heatmap-feature-layer.js",
   "dependencies": {
-    "leaflet": "^1.0.0-beta.1",
-    "esri-leaflet": "^2.0.0-beta.4",
+    "esri-leaflet": "^2.0.0",
+    "leaflet": "^1.0.0-rc.3",
     "leaflet.heat": "^0.1.3"
```

Comment:
can you bump leaflet.heat to `"^0.2.0"` please?
i noticed some erratic shifting of the heatmap on pan/zoom with the `1.x` version.